sunpy/physics/transforms/differential_rotation.py

```diff
@@ -401,3 +404,65 @@ def _sun_pos(date):
             "app_long": Longitude(l, u.deg),
             "obliq": Angle(oblt, u.deg)}
 
+
+def _to_norm(arr):
+    '''
+    Normalises an array. 
+    Needed for example for scikit-image which uses flotas between 0 and 1
+    '''
+    from skimage.util import img_as_float
+    arr = img_as_float(arr, force_copy=True)
+    if arr.min() < 0:
+        arr += np.abs(arr.min())
+    arr /= arr.max()
+    return arr
+
+def _un_norm(arr, ori):
+    '''
+    Un-normalises an array based in the original.
+    '''
+    level = 0 if ori.min() > 0 else np.abs(ori.min())
+    arr *= ori.max() + level
+    arr -= level
+    return arr   
+
+#Define a function that returns a new list of coordinates for each input coord.
+def _warp_sun(xy, smap, timedelta):
+
+    #Calculate the hpc coords
+    x = np.arange(0, smap.dimensions.x.value)
+    y = np.arange(0, smap.dimensions.y.value)
+    xx, yy = np.meshgrid(x, y)
+    hpc_coords = smap.pixel_to_data(xx * u.pix, yy * u.pix)
+    
+    #Do the diff rot
+    print smap.date, smap.date + timedelta
+    rotted = rot_hpc(hpc_coords[1], hpc_coords[0], smap.date,
+                     smap.date - timedelta, occultation=True)
+    # FIXME!! time is being subtracted so the output plot *seems* right
+    # but I don't understand it why ...
```

Comment:
oh, if I remember correctly you are defining the reverse coordinate transform, i.e. the desired coordinates to the original rather than the forward transform. A look at the scikit image docs would probably check my 2 year old memory.